extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -87,6 +87,7 @@ export const UIType = {
  *    consentId: ?string,
  *    currentPageId: string,
  *    currentPageIndex: number,
+ *    totalPages: number,
```

Comment:
Please prefer names that remove all doubts about what the variable contains. `totalPages` could be an array with all the pages, and `total` kind of means that it's the sum of something? I think something like `pagesCount` would be more obvious